### PR TITLE
[zh] Localize 1.18.6, 1.19.5, and 1.20.1 release notes

### PR DESCRIPTION
**Pull request by windsonsea**
- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/news/releases/1.20.x/announcing-1.20.1/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+title: 发布 Istio 1.20.1
+linktitle: 1.20.1
+subtitle: 补丁发布
+description: Istio 1.20.1 补丁发布。
+publishdate: 2023-12-12
+release: 1.20.1
+---
+
+本次发布修复了 12 月 12 日公布的 [`ISTIO-SECURITY-2023-005`](/zh/news/security/istio-security-2023-005)
+中阐述的安全更新和漏洞，提高了稳健性。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20.1/index.md:11]: 这个应该不是漏洞，就是普通的安全更新。

File: content/zh/news/releases/1.20.x/announcing-1.20.1/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: 发布 Istio 1.20.1
+linktitle: 1.20.1
+subtitle: 补丁发布
+description: Istio 1.20.1 补丁发布。
+publishdate: 2023-12-12
+release: 1.20.1
+---
+
+本次发布修复了 12 月 12 日公布的 [`ISTIO-SECURITY-2023-005`](/zh/news/security/istio-security-2023-005)
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20.1/index.md:10]:
```suggestion
本次发布实现了 12 月 12 日公布的安全更新 [`ISTIO-SECURITY-2023-005`](/zh/news/security/istio-security-2023-005)
```

File: content/zh/news/releases/1.20.x/announcing-1.20.1/index.md (hunk 1 above)
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20.1/index.md:11]:
```suggestion
以及一些错误修复，提高了稳健性。
```

File: content/zh/news/releases/1.20.x/announcing-1.20.1/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  ([Issue #47696](https://github.com/istio/istio/issues/47696))
+
+- **修复** 修复了由于 OpenShift 设置了 Pod 的 `SecurityContext.RunAs`
+  字段而导致在 OpenShift 上自定义注入 `istio-proxy` 容器时无法正常工作的问题。
+
+- **修复** 修复了在设置 `header-name: {}` 时 `VirtualService` 中的
+  HTTP 头存在匹配无法正常工作的问题。
+  ([Issue #47341](https://github.com/istio/istio/issues/47341))
+
+- **修复** 修复了多集群领导选举时不能让主领导优先从领导的问题。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20.1/index.md:32]:
```suggestion
- **修复** 修复了多集群领导选举时无法优先考虑本地而非远程领导者的问题。
```

File: content/zh/news/releases/1.19.x/announcing-1.19.5/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+本发布说明描述了 Istio 1.19.4 和 Istio 1.19.5 之间的不同之处。
+
+{{< relnote >}}
+
+## 变更 {#changes}
+
+- **修复** 修复 `istioctl tag set` 生成的 Webhook 意外被安装程序移除的问题。
+  ([Issue #47423](https://github.com/istio/istio/issues/47423))
+
+- **修复** 修复了多集群领导选举时不能让主领导优先从领导的问题。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.19.x/announcing-1.19.5/index.md:22]:
```suggestion
- **修复** 修复了多集群领导选举时无法优先考虑本地而非远程领导者的问题。
```

File: content/zh/news/releases/1.18.x/announcing-1.18.6/index.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+title: 发布 Istio 1.18.6
+linktitle: 1.18.6
+subtitle: 补丁发布
+description: Istio 1.18.6 补丁发布。
+publishdate: 2023-12-12
+release: 1.18.6
+---
+
+本次发布修复了 12 月 12 日公布的 [`ISTIO-SECURITY-2023-005`](/zh/news/security/istio-security-2023-005)
+中阐述的安全更新和漏洞，提高了稳健性。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.18.x/announcing-1.18.6/index.md:11]: 同 https://github.com/istio/istio.io/pull/14317/files#r1424766085，可以按建议修改下

File: content/zh/news/releases/1.19.x/announcing-1.19.5/index.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+title: 发布 Istio 1.19.5
+linktitle: 1.19.5
+subtitle: 补丁发布
+description: Istio 1.19.5 补丁发布。
+publishdate: 2023-12-12
+release: 1.19.5
+---
+
+本次发布修复了 12 月 12 日公布的 [`ISTIO-SECURITY-2023-005`](/zh/news/security/istio-security-2023-005)
+中阐述的安全更新和漏洞，提高了稳健性。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.19.x/announcing-1.19.5/index.md:11]: 同 https://github.com/istio/istio.io/pull/14317/files#r1424766085

File: content/zh/news/releases/1.18.x/announcing-1.18.6/index.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+  ([Issue #47505](https://github.com/istio/istio/issues/47505))
+
+- **修复** 修复了在设置 `header-name: {}` 时 `VirtualService` 中的
+  HTTP 头存在匹配无法正常工作的问题。
+  ([Issue #47341](https://github.com/istio/istio/issues/47341))
+
+- **修复** 修复了在没有追踪选项的情况下使用 `datadog` 或 `stackdriver` 时的空遍历问题。
+  ([Issue #45855](https://github.com/istio/istio/issues/45855))
+
+- **修复** 修复了多集群领导选举时不能让主领导优先从领导的问题。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.18.x/announcing-1.18.6/index.md:45]:
```suggestion
- **修复** 修复了多集群领导选举时无法优先考虑本地而非远程领导者的问题。
```

Comment (windsonsea) [reply content/zh/news/releases/1.18.x/announcing-1.18.6/index.md:45]: 这样会不会更通顺点：

```
- **修复** 修复了多集群领导者选举时未优先考虑本地领导者而考虑远程领导者的问题。
```

File: content/zh/news/releases/1.20.x/announcing-1.20.1/index.md (hunk 3 above)
Comment (windsonsea) [reply content/zh/news/releases/1.20.x/announcing-1.20.1/index.md:32]: `而非远程` 好像不通顺

File: content/zh/news/releases/1.18.x/announcing-1.18.6/index.md (hunk 7 above)
Comment (hanxiaop) [reply content/zh/news/releases/1.18.x/announcing-1.18.6/index.md]: 我觉得可以

Comment (windsonsea) [reply content/zh/news/releases/1.18.x/announcing-1.18.6/index.md]: 好的，那我 3 处都这么修改了

**Review by wilsonwu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**